### PR TITLE
feat(email-templates) - add email template in is own package

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: packages/email-templates/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# email-templates
```
Comment (searchableguy) [inline packages/email-templates/README.md:1]: Add more information and instructions please.

File: packages/email-templates/package.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "version": "1.0.0",
+  "name": "@unlock-protocol/email-templates",
```
Comment (searchableguy) [inline packages/email-templates/package.json:3]: You also need to handle this in root package.json since it is required to be built for CI use.

Comment (kalidiagne) [reply packages/email-templates/package.json:3]: added!

**Review by searchableguy — APPROVED** [review]
LGTM

**Pull request merged**